### PR TITLE
Ref Guide - Notifications

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10592

### CODE REVIEW

File: modules/reference/pages/home/home-notification-messages.adoc [hunk 1]
```diff
@@ -1,84 +1,42 @@
...
 = Notification Messages
 
-The menu:Main Menu[Home > Notification Messages] page allows you to manage your notification messages of the {productname} server.
+The menu:Home[Notification Messages] section shows all current messages produced by {productname}.
+By default, messages will remain current for thirty days.
+After this period, messages are deleted whether or not they are marked as read.
 
-image::home_notification_messages.png[scaledwidth=80%]
+To see unread messages, navigate to the [guimenu]``Unread Messages`` tab.
+For see all messages, navigate to the [guimenu]``All Messages`` tab.
```
Comment (keichwa) [inline modules/reference/pages/home/home-notification-messages.adoc]:
```suggestion
To see all messages, navigate to the [guimenu]``All Messages`` tab.
```
or adjust the previous sentence.

File: modules/reference/pages/home/home-notification-messages.adoc [hunk 2]
```diff
@@ -1,84 +1,42 @@
...
+[cols="1,1,1", options="header"]
+|===
+| Icon               | Description   | Example
+| icon:info-circle[role="none"] | Information | Client onboarding has failed.
+| icon:exclamation-triangle[role="orange"] | Warning | Channel synchronization has completed.
+| icon:times-circle[role="red"] | Error | Channel synchronization has failed.
+|===
+
+
+// needs to be moved to Admin or something, I think. LKB 2020-02-20
```
Comment (keichwa) [inline modules/reference/pages/home/home-notification-messages.adoc]: :+1:
the 30 days period was already mentioned at the beginning of this section.

File: modules/reference/pages/home/home-notification-messages.adoc [hunk 3]
```diff
@@ -1,84 +1,42 @@
...
 ----
 
 All notification types are enabled by default.
-To disable a notification type, open the [path]``/etc/rhn/rhn.conf`` configuration file and edit the [systemitem]``java.notifications_type_disabled`` parameter:
+To disable a notification type, add or edit this line in [path]``/etc/rhn/rhn.conf``:
 
 ----
-# Configure the disablement of notification messages by type - example disabling all notification types
 java.notifications_type_disabled = OnboardingFailed,ChannelSyncFailed,ChannelSyncFinished
 ----
```
Comment (keichwa) [inline modules/reference/pages/home/home-notification-messages.adoc]: Wondering why you removed the rest.  But ok.

Comment (jcayouette) [reply modules/reference/pages/home/home-notification-messages.adoc]: Agreed, is there a reason we are removing the legend? It probably needs some cleanup but this is a large chunk.

Comment (Loquacity) [reply modules/reference/pages/home/home-notification-messages.adoc]: The legend is now in the table.

**Comment by Loquacity** [comment]
> LGTM, although I wouldn't remove the legend.

See my comment. It wasn't deleted.

File: modules/reference/pages/home/home-notification-messages.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/reference/pages/home/home-notification-messages.adoc]: https://github.com/SUSE/spacewalk/issues/10794

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> The legend is now in the table.

Thanks you!

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
Product & Solution Documentation   Maxfeldstraße 5
                                   90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Pull request merged**